mlflow/fastai.py

```diff
@@ -90,6 +111,13 @@ def save_model(fastai_learner, path, conda_env=None, mlflow_model=Model(), **kwa
     model_data_path = Path(model_data_path)
     os.makedirs(path)
 
+    if mlflow_model is None:
+        mlflow_model = Model()
+    if signature is not None:
+        mlflow_model.signature = signature
+    if input_example is not None:
+        save_example(mlflow_model, input_example, path)
+
     # Save an Learner
     fastai_learner.export(model_data_path, **kwargs)
```

Comment:
No action needed, just a note that I confirmed that `signature` & `input_example` are not valid arguments to [export](https://docs.fast.ai/basic_train.html#Learner.export)